crates/ruff/src/rules/pep8_naming/settings.rs

```diff
@@ -1,9 +1,14 @@
 //! Settings for the `pep8-naming` plugin.
 
+use std::error::Error;
+use std::fmt;
+
 use serde::{Deserialize, Serialize};
 
 use ruff_macros::{CacheKey, CombineOptions, ConfigurationOptions};
 
+use crate::settings::types::IdentifierPattern;
+
 const IGNORE_NAMES: [&str; 12] = [
```

Comment:
Nit: can we amend the documentation (`/// A list of names to ignore when considering `pep8-naming` violations.`, below) to mention that globs are supported?